test/fixtures/api.js

```diff
@@ -511,6 +511,56 @@ module.exports = {
       }],
     }),
 
+  getBranchListNoOriginSet: (envSecret = 'forestEnvSecret', haveCurrent = true) => nock('http://localhost:3001')
```

Comment:
you do not use `haveCurrent` you can simply remove it IMO
+ you can reduce the fixture by keeping only the branch who does not have origin

Reply:
done !